tests/integration/test_client.py

```diff
@@ -57,3 +61,49 @@ async def test_client_start_transcribe_stream_bad_request(self, client):
                 media_sample_rate_hz=9999999,
                 media_encoding="pcm",
             )
+
+    @pytest.mark.asyncio
+    async def test_client_start_transcribe_stream_incompatible_args(self, client):
+        # show_speaker_label and enable_channel_identification should
+        # not be both set when making StartStreamTranscriptionRequest.
+        # Test that exception raised by service.
+        with pytest.raises(BadRequestException):
+            stream = await client.start_stream_transcription(
+                language_code="en-US",
+                media_sample_rate_hz=16000,
+                media_encoding="pcm",
+                show_speaker_label="true",
+                enable_channel_identification="true",
+                number_of_channels=2,
+            )
+
+    @pytest.mark.asyncio
+    async def test_start_transcribe_stream_bad_boolean_show_speaker_lab(self, client):
+        with pytest.raises(SerializationException):
+            stream = await client.start_stream_transcription(
+                language_code="en-US",
+                media_sample_rate_hz=16000,
+                media_encoding="pcm",
+                show_speaker_label="foo",
+                enable_channel_identification="true",
+            )
+        with pytest.raises(SerializationException):
```

Comment:
Sorry, I meant we should just get rid of this entirely. We don't need to validate every parameter from the service is working correctly on their end. We just want to make sure that generally if a bad boolean-string is passed to them, we're getting the SerializationException.